### PR TITLE
Make runtime display fallback when needed.

### PR DESCRIPTION
**Pull request by sriramkrish85**
Closes https://github.com/ampproject/amphtml/issues/2261

### CODE REVIEW

File: src/custom-element.js [hunk 1]
```diff
@@ -934,6 +934,10 @@ export function createAmpElementProto(win, name, implementationClass) {
     // element, (b) some realyout is expected and (c) fallback condition would
     // be rare.
     this.classList.toggle('amp-notsupported', state);
+    const fallbackElement = this.getFallback();
```
Comment (dvoytenko) [inline src/custom-element.js]: Please only search for fallback element when `state == true`

Comment (sriramkrish85) [inline src/custom-element.js]: Done

File: src/custom-element.js [hunk 2]
```diff
@@ -934,6 +934,10 @@ export function createAmpElementProto(win, name, implementationClass) {
...
+    if (state && fallbackElement) {
+      this.resources_.scheduleLayout(this, fallbackElement);
```
Comment (sriramkrish85) [inline src/custom-element.js]: yes - added that in the manual file - and updated the unit test.

**Comment by sriramkrish85** [comment]
PTAL

**Comment by dvoytenko** [comment]
LGTM

**Pull request merged**